README.md

```diff
@@ -1,2 +1,5 @@
 # n-teaser
-n-card and o-card's replacement - better styles, simpler combinations &amp; a more basic templating
+n-card and o-card's replacement - better styles, simpler combinations &amp; more basic templating
```

Comment:
`n-teaser` will only replace `n-card`, with `o-teaser` replacing `o-card` (aim is to have styles in `o-teaser` and leave templating & logic to `n-teaser`)

Less an issue, just an FYI